src/sentry/api/endpoints/organization_releases.py

```diff
@@ -411,5 +413,13 @@ def post(self, request, organization):
             else:
                 status = 201
 
+            analytics.record(
```

Comment:
@NisanthanNanthakumar The releases endpoint is idempotent in that you can call this endpoint many times to create a particular release and get a 2xx status code but it will only create one. We actually have logic above for returning a `201` or `208` status code. We should either record that information in this event or only generate the event when we are actually creating the release. Same with the project releases endpoint.

Reply:
@scefali Yes, we are recording that information under `created_status=status` on line 422